development.postgresql.yml

```diff
@@ -0,0 +1,72 @@
+version: "2"
+services:
+  server:
+    build:
+      context: .
+      dockerfile: server.Dockerfile
+    networks:
+      serverdb:
+      signer:
+      server:
+        aliases:
+          - notary-server
+    entrypoint: /usr/bin/env sh
+    command: -c "./migrations/migrate.sh && notary-server -config=fixtures/server-config.postgres.json"
+    environment:
+      MIGRATIONS_PATH: migrations/server/postgresql
+      DB_URL: postgres://server@serverdb:5432/notaryserver?sslmode=disable
+    depends_on:
+      - serverdb
+      - signer
+  signer:
+    build:
+      context: .
+      dockerfile: signer.Dockerfile
+    networks:
+      signerdb:
+      signer:
+        aliases:
+          - notarysigner
+    entrypoint: /usr/bin/env sh
+    command: -c "./migrations/migrate.sh && notary-signer -config=fixtures/signer-config.postgres.json"
+    environment:
+      MIGRATIONS_PATH: migrations/signer/postgresql
+      DB_URL: postgres://signer@signerdb:5432/notarysigner?sslmode=disable
+    depends_on:
+      - signerdb
+  serverdb:
```

Comment:
using single image also means that we need to bootstrap two different database with extra scripts similar to `notarymysql`. But the CHANGELOG also stated that:

```
+ Update `docker-compose` configuration to use official mariadb image
    + deprecate `notarymysql`
    + default to using a volume for `data` directory
    + use separate databases for `notary-server` and `notary-signer` with separate users
```

I can add extra bootstrap scripts, but i just think its cleaner this way. I also dont think commiting two databases are much harder than one.

But IANAM, Let me know what you think.